test/Makefile

```diff
@@ -0,0 +1,28 @@
+# http://clarkgrubb.com/makefile-style-guide
+
+MAKEFLAGS += --warn-undefined-variables --no-print-directory
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := test
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+tag = claranet/direnv-asdf
+
+.PHONY: test
+test:
+	mkdir -p .direnv .docker/.direnv .docker/home
+	docker run --rm \
+		-v $(PWD):/src \
```

Comment:
Mount the current directory (would commonly be the top of the repo) at `/src` or anywhere you want, as long as you are consistent with the subsequent lines.